src/components/app/tour-list/TourListActions.tsx

```diff
@@ -9,7 +9,7 @@ import { Link } from 'react-router-dom'
 type TourListActionsProps = {
   id: string
 }
-export default function TourListActions ({ id }: TourListActionsProps): JSX.Element {
+const TourListActions = ({ id }: TourListActionsProps): JSX.Element => {
```

Comment:
Remove return type hint as it's a component.

(I think that's my component from before our styleguide definition... sorry :D)